Sources/PrimerSDK/Classes/PCI/Checkout Components/PrimerCardRedirectData.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  PrimerCardRedirectData.swift
+//  PrimerSDK
+//
+//  Created by Dario Carlomagno on 27/09/22.
+//
+
+#if canImport(UIKit)
+
+import Foundation
+
+public class PrimerCardRedirectData: PrimerRawData {}
+
+public class PrimerBancontactCardRedirectData: PrimerCardRedirectData {
```

Comment:
This could be extended by `PrimerCardData`, then `PrimerCardData` would only have the cvv. This is just a suggestion for the future.

Reply:
🙏 yeah I get the idea - thanks!